Sem. 03/Pract. 03/Task01.cpp

```diff
@@ -0,0 +1,197 @@
+#include <iostream>
+#include <fstream>
+
+const char* OUTPUT_FILE_NAME = "sorted.bin";
+const char* FILE_NAME = "orders.txt";
+
+const int MAX_SIZE = 25;
+
+struct Order {
+    char address[MAX_SIZE];
+    double price;
+};
+
+void saveOrder(std::ofstream& file, const Order& order) {
+    file.write((const char*) order.address, MAX_SIZE);
+    file.write((const char*) &order.price, sizeof(double));
+}
```

Comment:
this way of writing objects to file will only be used when we have objects with varying memory